v-next/core/src/index.ts

```diff
@@ -28,4 +28,4 @@ export async function createHardhatRuntimeEnvironment(
 }
 
 export { resolvePluginList } from "./internal/plugins/resolve-plugin-list.js";
-export { buildGlobalParameterMap } from "./internal/global-parameters.js";
+export { buildGlobalParametersMap } from "./internal/global-parameters.js";
```

Comment:
@schaable  We already discussed changing this name in a future PR, but just FYI, this was correct. I It can be tricky for us, native Spanish speakers, but it's ParameterMap, not ParametsMap. You have to keep the first word in singular, even though the name represents a collection of those things. Pretty much the contrary to what you'd do in Spanish.